docs/docs/tutorials/creating-fake-user-data.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Hands-On Tutorial - Creating fake user data in GA to train your number-generating skills 
+image: https://storage.googleapis.com/getsynth-public/media/users.jpg
+description: In this tutorial we're going to create fake GA data with synth!
+---
+
+![Logs](img/users.jpg)
+
+In this tutorial, we will show you that even if the target data looks tricky at first glance, with [`synth`](http://getsynth.com), there is always a solution. We will focus on numeric data.
+
+Together, we will create fake Google Analytics data imitating the behavior of individual website users at the session level, also known as [User Explorer](https://support.google.com/analytics/answer/6339208?hl=en#zippy=%2Cin-this-article). 
+
+For those who actually work with GA: this kind of data may help your data science team in testing user profiling algorithms for e-commerce or online campaigning.
+
+## Prerequisite: Install Synth
+If you have not yet done so, you need to get `synth` running on your machine. It can be installed with a single line from a CLI tool of your choice as shown [here](https://getsynth.com/download).
+
+## Elaborating a Data Model
+
+In the GA GUI, a target report would like like this:
+| User ID      | Sessions | Avg. Session Duration | Bounce rate | Revenue | Transactions | Goal Conversion Rate
+| ----------- | ----------- | ----------- | ----------- | ----------- | ----------- | ----------- |
+| fcnMA33      | 64       |  00:06:43 | 1.56% | $0.00 | 0 | 96.88% |
+| LAhfn66   | 48        | 00:07:59 | 2.08% | $0.00 | 0 | 33.33% |
+
+Before we start writing a code, let's look at the data closely. 
+
+In our case, in the table we see values in different formats. In reality though, we can limit them to just two data types: integers and floating numbers.
+
+Integer columns are:
+* Sessions
+* Transactions 
+
+Floating type columns are:
+* Bounce rate 
+* Avg. Session Duration
+* Revenue
+* Goal Conversion Rate
+
+For all columns, we need numbers that are equal or greater than zero. Besides, for the percentage columns, we need to introduce the upper limit. The duration column will be expressed in minutes since it is a bit more intuitive than hours and seconds. Additionally, we need to specify a reasonable maximum duration as well. 
+
+---
+**note**
```

Comment:
You may want to use [Docusaurus admonitions](https://docusaurus.io/docs/markdown-features/admonitions) for this, as it integrates better in the rendered version.